homeassistant/components/nissan_leaf/device_tracker.py

```diff
@@ -15,28 +15,25 @@
 
 def setup_scanner(hass, config, see, discovery_info=None):
     """Set up the Nissan Leaf tracker."""
-    _LOGGER.debug("Setting up Scanner (device_tracker) for Nissan Leaf, "
-                  "discovery_info=%s", discovery_info)
-
     def see_vehicle():
```

Comment:
Add a guard clause here that checks if discovery_info is None and return False if so.

Put the guard clause directly first in `setup_scanner` body.

Reply:
Fixed with commit https://github.com/home-assistant/home-assistant/pull/21145/commits/f60f441a87633840be6f59168c6e18972f8a75c5